### PR TITLE
Type definition generator information to readme

### PR DESCRIPTION
**Pull request by egekorkan**
I had to look up into the package.json files to find and I think it can be mentioned in a more visible place.

### CODE REVIEW

**Review by relu91 — APPROVED** [review]
Good point, thanks!

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ If you want to make a full text review on the spec or other files, follow the st
 ## TypeScript Definitions
 
 The specification uses WebIDL definitions, but [TypeScript definititions](./typescript) are also available.
+We use <https://www.npmjs.com/package/json-schema-to-typescript> to generate the TypeScript definitions.
```
Comment (danielpeintner) [inline README.md:22]:
```suggestion
We use <https://www.npmjs.com/package/json-schema-to-typescript> to generate the TypeScript definitions for [wot-thing-description-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-description) and [wot-thing-models-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-model).
```

Maybe we can be more explicit ..

Comment (danielpeintner) [reply README.md:22]: Note, personally I also do not like to link like 
<https://www.npmjs.com/package/json-schema-to-typescript>

I would prefer to do it like
[json-schema-to-typescript](https://www.npmjs.com/package/json-schema-to-typescript)

File: README.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ If you want to make a full text review on the spec or other files, follow the st
 ## TypeScript Definitions
 
 The specification uses WebIDL definitions, but [TypeScript definititions](./typescript) are also available.
+We use <https://www.npmjs.com/package/json-schema-to-typescript> to generate the TypeScript definitions for [wot-thing-description-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-description) and [wot-thing-models-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-model).
```
Comment (egekorkan) [inline README.md:22]:
```suggestion
We use [json-schema-to-typescript](https://www.npmjs.com/package/json-schema-to-typescript) to generate the TypeScript definitions for [wot-thing-description-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-description) and [wot-thing-models-types](https://github.com/w3c/wot-scripting-api/tree/main/typescript/thing-model).
```

**Comment by egekorkan** [comment]
@danielpeintner I have applied your suggestion and changed the linking style

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**